flaskapp/view_functions/ivrflow.py

```diff
@@ -29,23 +29,25 @@ def voice_joined():
     else:
         resp.say(f'We got your answer {answer}. We hope you will back us later. Take care.')
         resp.hangup()
-    return(str(resp))
+    return (str(resp))
+
 
 def voice():
     """ Function for answering from any call to Main Number of the IVR """
     resp = VoiceResponse()
     tel = request.values['From']
     user = check_new_user(tel)
-    if user == 'Exist':resp.dial(optional_number)
+    if user == 'Exist':
+        resp.dial(optional_number)
     else:
-        save_new_user(tel,'Calls')
-        gather = Gather(input='speech dtmf', action='/voice_joined',  timeout=3, num_digits=1)
-        gather.say('Welcome to Heart Voices ! We help people change their habits on their way to a healthy life without heart disease. Do you want to join us? Say yes or no.')
+        save_new_user(tel, 'Calls')
+        gather = Gather(input='speech dtmf', action='/voice_joined', timeout=3, num_digits=1)
+        gather.say(
+            'Welcome to Heart Voices ! We help people change their habits on their way to a healthy life without heart disease. Do you want to join us? Say yes or no.')
```

Comment:
All long sentences I would put into separate file, e.g. `conversations.py` or  `dialogs.py` with the following structure, e.g.:
# ---  dialogs.py
GREETINGS = \
"""
Welcome to Heart Voices ! We help people change their habits on their way to a healthy ....

"""
As a result, the code would be as follows: `gather.say(GREETINGS)`, and somewhere in the top of `ivrflow` we had `from dialogs import GREETINGS, etc..`

Reply:
done